apps/builder/app/shared/copy-paste/plugin-webflow/plugin-webflow.ts

```diff
@@ -118,25 +118,53 @@ const parse = (clipboardData: string) => {
     return;
   }
 
-  const unsupportedNodeTypes: Array<string> = data.payload.nodes
-    .filter((node: { type: string }) => {
-      return (
-        node.type !== undefined &&
-        wfNodeTypes.includes(node.type as (typeof wfNodeTypes)[number]) ===
-          false
-      );
-    })
-    .map((node: { type: string }) => node.type);
-
-  if (unsupportedNodeTypes.length !== 0) {
-    const message = `Skipping unsupported elements: ${unsupportedNodeTypes.join(", ")}`;
+  const unsupportedNodeTypes: Set<string> = new Set(
+    data.payload.nodes
+      .filter((node: { type: string }) => {
+        return (
+          node.type !== undefined &&
+          wfNodeTypes.includes(node.type as (typeof wfNodeTypes)[number]) ===
+            false
+        );
+      })
+      .map((node: { type: string }) => node.type)
+  );
+
+  if (unsupportedNodeTypes.size !== 0) {
+    const message = `Skipping unsupported elements: ${[...unsupportedNodeTypes.values()].join(", ")}`;
     toast.info(message);
     console.info(message);
   }
 
   const result = WfData.safeParse(data);
 
   if (result.success) {
+    const unpasedTypes = new Set<string>();
+    // console.log("result.data.payload.nodes", result.data.payload.nodes);
+    for (let i = 0; i !== result.data.payload.nodes.length; ++i) {
```

Comment:
can we add a linter for 1 char variables?

Reply:
Better no, it breaks x, y, z axis and p, b, I tags which are perfectly valid